src/Task/FileSystem/CopyDir.php

```diff
@@ -26,13 +29,39 @@ public function run()
         return Result::success($this);
     }
 
+    /**
+     * Sets the default folder permissions for the destination if it doesn't exist
+     *
+     * @link http://en.wikipedia.org/wiki/Chmod
+     * @link http://php.net/manual/en/function.mkdir.php
+     * @link http://php.net/manual/en/function.chmod.php
+     * @param int $value
+     * @return $this
+     */
+    public function chmod($value)
+    {
+        $this->chmod = (int)$value;
+        return $this;
+    }
+
+    /**
+     * Copies a directory to another location.
+     *
+     * @param string $src Source directory
+     * @param string $dst Destination directory
+     * @throws \Robo\Exception\TaskException
+     * @return void
+     */
     protected function copyDir($src, $dst)
     {
         $dir = @opendir($src);
         if (false === $dir) {
             throw new TaskException($this, "Cannot open source directory '" . $src . "'");
         }
-        @mkdir($dst);
+        @mkdir($dst, $this->chmod);
```

Comment:
This line was the cause of the error in #125 ...

Please see https://github.com/Codegyre/Robo/pull/126#issuecomment-78880801